data/tsi.src

```diff
@@ -92343,6 +92343,7 @@
 狗尾續貂 0 ㄍㄡˇ ㄨㄟˇ ㄒㄩˋ ㄉㄧㄠ
 狗屁 26 ㄍㄡˇ ㄆㄧˋ
 狗屁不通 5 ㄍㄡˇ ㄆㄧˋ ㄅㄨˋ ㄊㄨㄥ
+狗屁倒灶 10 ㄍㄡˇ ㄆㄧˋ ㄉㄠˇ ㄗㄠˋ
```

Comment:
The subject uses `ㄆㄧˊ` while the body takes "ㄆㄧˋ", that is inconsistent. Please use `git commit --amend` to rework.